src/utils/index.ts

```diff
@@ -2,3 +2,22 @@ import clsx, { type ClassValue } from "clsx";
 import { twMerge } from "tailwind-merge";
 
 export const cn = (...classes: ClassValue[]) => twMerge(clsx(...classes));
+
+export const formatDate = (isoDate: string): string => {
+  const date = new Date(isoDate);
+  return new Intl.DateTimeFormat("pl-PL", {
+    day: "2-digit",
+    month: "2-digit",
+    year: "numeric",
+  }).format(date);
+};
+
+export const cleanHTML = (html: string): string => {
+  return html.replace(/<[^>]+>/g, "").trim();
```

Comment:
Could be implicitly returned. Tho depends on the preferences - could even be enforced by the linter rules for the project.